modules/light-clients/08-wasm/keeper/genesis_test.go

```diff
@@ -70,7 +71,7 @@ func (suite *KeeperTestSuite) TestExportGenesis() {
 	suite.SetupTest()
 	ctx := suite.chainA.GetContext()
 
-	expCodeHash := "c64f75091a6195b036f472cd8c9f19a56780b9eac3c3de7ced0ec2e29e985b64"
+	expCodeHash := "561715ea6ee1dce8f78499914e6c7853dc315a5e6ecf01da09a9054a160e5d1d"
```

Comment:
any idea where these magic values come from?

Reply:
Yes, this is the hash of the tendermint contract that we have in the test data folder.